workbench-core/accounts/src/models/accountSchema.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
```

Comment:
why are we using zod here?

Reply:
For runtime and transpile time type checking and data validation.